inlong-sort/sort-connectors/pom.xml

```diff
@@ -113,6 +118,13 @@
             <groupId>org.apache.pulsar</groupId>
             <artifactId>pulsar-client-all</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.streamnative.connectors</groupId>
+            <artifactId>pulsar-flink-connector_2.11</artifactId>
+            <version>1.13.6.1-rc9</version>
```

Comment:
Please put the version number into the pom file of the root path, thanks.

Reply:
put it in root pom.xml